.github/workflows/main.yml

```diff
@@ -2,40 +2,287 @@ name: CI
 
 on:
   push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - master
+# TODO: restore before merging
+#    branches:
+#      - master
+#  pull_request:
+#    branches:
+#      - master
+
+env:
+ DIST_ROOT: '/ipns/dist.ipfs.io' # content root used for calculating diff to build
+ GO_IPFS_VER: 'v0.9.1'           # go-ipfs daemon used for chunking and applying diff
+ CLUSTER_CTL_VER: 'v0.14.0'      # ipfs-cluster-ctl used for pinning
```

Comment:
What's the process for updating these?  Do we rely on someone manually updating it?  What's their trigger for doing so?

Reply:
I think those are low importance: could be kept as-is until someone cares to update or there is a breaking change that forces version bump.

Having static versions of build tools makes the builds more robust and reproducible, but we could also switch to always run on the latest version for dogfooding – don't feel strongly either way, but leaning towards more reproducibility.